browser/brave_webcompat_reporter/webcompat_reporter_service.cc

```diff
@@ -0,0 +1,102 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_webcompat_reporter/webcompat_reporter_service.h"
+
+#include <string>
+
+#include "net/base/load_flags.h"
+#include "content/public/browser/browser_task_traits.h"
+#include "base/task/post_task.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+#include "chrome/browser/browser_process.h"
```

Comment:
see below to get rid of this include and then this should move to `components/webcompat_reporter/browser/`